api/chaos/experimentation/v1/experimentation.proto

```diff
@@ -41,8 +41,14 @@ message CreateExperimentResponse {
   Experiment experiment = 1;
 }
 
+enum GetExperimentsStatus {
```

Comment:
Not sure why do we have Status here and in Experiment Run details? Also, why do we only have `running` and not other statuses? Can we not have a generic status and inherit it in both places if we need?

Reply:
Because for quering we support only 2 statuses - `Running` and `Unspecfied` - and for returning the details of an experiment we do support 6 of them. If I wanted to use the same enum in both places I would need to implement more querying logic that we currently need + I do not think that we should be reusing the same `Status` types for quering and returning the results of quering since it's not clear for me whether these 2 will ever become the same.